docs/language/sdk/display.mdx

```diff
@@ -189,6 +200,17 @@ that updates the scene as new data arrives. Here is a bigger example that
 subscribes to an MQTT topic for its weather information, and uses the
 local time to update a digital clock on the same display.
 
+This example also shows the use of a style built up with maps, to let
+multiple elements be styled with the same parameters.  We can use
+the syntax `--id="my-name"` to given an element a name, much like
+the HTML syntax `<div id="my-name">`.  Similarly, we can give an element a
+class with `--classes=["my-class"]`, which is analogous to
+the HTML syntax `<div class="my-class">`.  We don't use `--class` because
+that is a reserved keyword in Toit.
```

Comment:
I wouldn't justify `classes` vs `class`. Since there can be more than one, `classes` is actually "righter".
```suggestion
the HTML syntax `<div class="my-class">`.
```